src/percy-agent-client/percy-agent.ts

```diff
@@ -84,17 +84,20 @@ export default class PercyAgent {
 
   private serializeInputElements(doc: HTMLDocument): HTMLDocument {
     const domClone = doc.documentElement
-    const inputNodes = domClone.getElementsByTagName('input')
-    const inputElements = Array.prototype.slice.call(inputNodes) as HTMLInputElement[]
+    const formNodes = domClone.querySelectorAll('input, textarea')
+    const formElements = Array.prototype.slice.call(formNodes)
 
-    inputElements.forEach((elem: HTMLInputElement) => {
+    formElements.forEach((elem: HTMLInputElement) => {
       switch (elem.type) {
       case 'checkbox':
       case 'radio':
         if (elem.checked) {
           elem.setAttribute('checked', '')
         }
         break
+      case 'textarea':
+        // setting text or value does not work but innerText does
+        elem.innerText = elem.value
```

Comment:
Any of these will work:

- `elem.innerText = elem.value`
- `elem.value = elem.value`
- `elem.textContent = elem.value`